README.md

```diff
@@ -27,55 +27,73 @@ The pipeline, constructed using the `nf-core` [template](https://nf-co.re/tools#
 
-3. Extract Target Reads
+   - This step generates consensus sequences for a large number of reads mapped to pathogen genomes using [samtools](<(http://www.htslib.org/)>) for Illumina reads or [medaka](https://github.com/nanoporetech/medaka) for Nanopore reads. The resulting consensus sequence will then be used as input for BLAST.
+
+3. **BLAST for pathogen identification**
+
+   - Use `BLAST` to identify the closest reference genomes for the target reads. There are two options: `BLASTx` using [DIAMOND](https://github.com/bbuchfink/diamond) based on a protein database, and [BLASTn](https://blast.ncbi.nlm.nih.gov/Blast.cgi) based on a nucleotide database.
+
+4. **Extract target reads**
 
    - From the mapped reads, extract the target reads that match the predefined viral pathogens based on the result of `BLAST`.
 
-4. Visualize Using IGV
+5. **Visualize using IGV**
 
    - Visualize the extracted reads using `IGV` (Integrative Genomics Viewer) to provide a graphical representation for detailed analysis.
 
-5. Perform Quality Check
-   - Conduct quality checks on the target reads using `FASTQC` and `MultiQC` to ensure data quality and reliability.
+6. **Perform quality check**
+   - Conduct quality checks on the target reads using [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and [MultiQC](<(http://multiqc.info/)>) to ensure data quality and reliability.
+
+### Orange Workflow - Verify Identified Viruses
+
+This workflow is activated by enabling the `--perform_extract_reads` option and disabling the `--taxid`.
 
-### Orange Pipeline - Verify Identified Viruses
+1. **Decontamination**
 
-1. Extract Viral TaxIDs
+   - Filter the output files from metagenomics classifiers like [Kraken2](https://ccb.jhu.edu/software/kraken2/), [Centrifuge](https://ccb.jhu.edu/software/centrifuge/), or [DIAMOND](https://github.com/bbuchfink/diamond) to remove false positives and background contaminations. This step compares results to the negative control and identifies likely present species based on user-defined thresholds.
 
-   - Extract viral TaxIDs predicted by taxonomic classification tools such as `Kraken2`, `Centrifuge`, and `DIAMOND`.
+2. **Extract viral TaxIDs**
 
-2. Extract Classified Reads
+   - Extract **_viral_** TaxIDs predicted by taxonomic classification tools such as `Kraken2`, `Centrifuge`, and `DIAMOND`.
 
-   - Extract the reads classified as viruses based on the identified TaxIDs.
+3. **Extract reads**
 
-3. BLAST
+   - Extract the reads classified as viruses based on a list of identified TaxIDs.
 
-   - Identify the closest reference genomes for the extracted reads using `BLAST`.
+4. **de-novo assembly**
 
-4. Mapping
+   - This step performs de-novo assembly for TaxIDs containing a large number of reads. [Spades](https://github.com/ablab/spades) is used for Illumina reads, and [Flye](https://github.com/mikolmogorov/Flye) is used for Nanopore reads. The resulting contig files will be used as input for `BLAST`.
```

Comment:
What do you mean here with `large number of reads`?

Reply:
What I basically meant was that de-novo shouldn't be run on too few reads.